install_config/install/advanced_install.adoc

```diff
@@ -289,7 +289,7 @@ options] in the OAuth configuration. See xref:advanced-install-session-options[C
 |`openshift_master_session_encryption_secrets`
 
 |`openshift_set_node_ip`
-|This variable configures `nodeIP` in the node configuration. This variable is needed in cases where it is desired for node traffic to go over an interface other than the default network interface. The host variable `openshift_ip` can also be configured on each node.
+|This variable configures `nodeIP` in the node configuration. This variable is needed in cases where it is desired for node traffic to go over an interface other than the default network interface. The host variable `openshift_ip` can also be configured on each node to set a specific IP that might not be the IP of the default route. `openshift_ip` can be used for etcd.
```

Comment:
I think we should move the tidbit about `openshift_ip` influencing etcd configuration closer to the `openshift_ip` variable block since `openshift_set_node_ip` is only relevant to the node service.